blocks/form/form.js

```diff
@@ -56,8 +56,16 @@ function createButton(field) {
       if (checkValidity(form) !== null) {
         button.setAttribute('disabled', '');
         await submitForm(form);
-        const redirectTo = field.Extra;
-        window.location.href = redirectTo;
+        const formContainer = form.closest('.form-container');
+        formContainer.firstElementChild.classList.add('hide');
+        form.classList.add('hide');
+        const paragraph = document.createElement('p');
+        paragraph.classList.add('contact-us');
+        const text = document.createTextNode(
```

Comment:
Thinking about the Franklin author, can this text be brought into the Word doc where this block is defined?

Thinking ahead to when XD comes back with a more elaborate design beyond a simple string.

Please log a Jira if this will take a bit more effort.

Reply:
Since it is a form, maybe we could add it to the email-form spreadsheet. Currently we pick up the form labels from there. So, it would make sense to add it to email-form.xls. I'll give it a shot now.

Reply:
Done.